dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -2285,7 +2287,8 @@ protected void killVM(String vmDir, String clientName, boolean isLead) {
         Log.getLogWriter().info(clientName + " stopped successfully...");
     }
 
-    protected void startVM(String vmDir, String clientName, boolean isLead) {
+    protected void startVM(String vmDir, String clientName, boolean isLead) throws InterruptedException {
+        Thread.sleep(100000);
```

Comment:
We had a discussion about the SNAP-1177 and at that time, you had suggested to make sure of restarting the member after certain delay. That's the reason, I added this delay of 100 secs before calling start for the member which is getting recycled.